src/mail/view/MailViewerViewModel.ts

```diff
@@ -909,7 +909,17 @@ export class MailViewerViewModel {
 	}
 
 	async downloadAll(): Promise<void> {
-		await this.fileController.downloadAll(this.getNonInlineAttachments())
+		try {
+			await this.fileController.downloadAll(this.getNonInlineAttachments())
+		} catch (e) {
+			if (e instanceof FileOpenError) {
+				console.warn("FileOpenError", e)
+				await Dialog.message("canNotOpenFileOnDevice_msg") // FIXME extra error message?
```

Comment:
noting FIXME for later

Reply:
so are we going to keep it like this for now?